File: java/calendar/src/test/java/com/rbleggi/calendar/UserTest.java
package com.rbleggi.calendar;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class UserTest {
    @Test
    void userHasCorrectId() {
        var user = new User("1", "Alice");
        assertEquals("1", user.id());
    }

    @Test
    void userHasCorrectName() {
        var user = new User("1", "Alice");
        assertEquals("Alice", user.name());
    }
}

File: kotlin/calendar/src/test/kotlin/com/rbleggi/calendar/UserTest.kt
package com.rbleggi.calendar

import kotlin.test.Test
import kotlin.test.assertEquals

class UserTest {
    @Test
    fun `user has correct id`() {
        val user = User("1", "Alice")
        assertEquals("1", user.id)
    }

    @Test
    fun `user has correct name`() {
        val user = User("1", "Alice")
        assertEquals("Alice", user.name)
    }
}
